com.transmissionbt.Transmission.json

```diff
@@ -178,6 +178,45 @@
                 }
             ]
         },
+        {
+            "name": "canberra",
+            "cleanup": [
+                "/etc",
+                "/include",
+                "/lib/gnome-settings-daemon-3.0",
+                "/lib/gtk-3.0",
+                "/libexec",
+                "/share/gtk-doc",
+                "/share/vala"
+            ],
+            "config-opts": [
+                "--disable-static",
+                "--disable-gtk-doc",
+                "--disable-oss",
+                "--enable-pulse",
+                "--disable-udev",
+                "--disable-gtk",
+                "--enable-gtk3",
+                "--disable-lynx"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://0pointer.de/lennart/projects/libcanberra/libcanberra-0.30.tar.xz",
+                    "sha256": "c2b671e67e0c288a69fc33dc1b6f1b534d07882c2aceed37004bf48c601afa72"
+                }
+            ]
+        },
```

Comment:
```suggestion
        "shared-modules/libcanberra/libcanberra.json",
```

Reply:
How do I keep `/bin/canberra-gtk-play`? The one in share-module cleans up `/bin`

Reply:
Oh, I missed that detail. Sorry.